components/layout_2020/table/layout.rs

```diff
@@ -2025,7 +1984,37 @@ impl<'a> TableLayout<'a> {
             CollapsedBlockMargins::zero(),
         )
         .with_baselines(baselines)
-        .with_detailed_layout_info(detailed_layout_info)
+        .with_detailed_layout_info(self.detailed_layout_info_for_grid())
+    }
+
+    fn detailed_layout_info_for_grid(&mut self) -> Option<SpecificLayoutInfo> {
```

Comment:
```suggestion
    fn specific_layout_info_for_grid(&mut self) -> Option<SpecificLayoutInfo> {
```

Reply:
We need to have a consistent terminology because `BoxFragment` has `with_detailed_layout_info()` but the type is `SpecificLayoutInfo`. #35078

Reply:
I think we just need to use specific_layout_info everywhere. The original name was `DetailedLayoutInfo`. In review the struct was renamed, but not all of the functions that dealt with it.